packages/remediations/src/RemediationWizard/RemediationWizard.js

```diff
@@ -97,7 +99,7 @@ const RemediationWizard = ({
 
     const validatorMapper = {
         'validate-systems': () => (value) => (
-            value && value.length > 0
+            value && Object.keys(value).length > 0
```

Comment:
Here I would probably look at `Object.values(value).filter(value => typeof value !== undefined).length`. If its possible that value can be `undefined` or `null` or whatever is the value after `select value -> de select it -> select other one`.